docs/options-reference.md

```diff
@@ -62,6 +62,10 @@ Here are all the options available when configuring the module and their default
     // preferred language to prevent subsequent redirections
     // Set to false to redirect every time
     useCookie: true,
+    // Enabled by default, which means the cookie domain includes the subdomain
+    // (e.g. "www.mydomain.com")
+    // If disabled, the subdomain is excluded (e.g. "mydomain.com")
+    includeCookieSubdomain: true
```

Comment:
I'm not sure about the naming since what would it mean in the case when the page is on `mydomain.com`? Then one would think that this option is irrelevant while it's not the case - toggling it to false would still change behavior as the behavior of cookie with explicit domain set is different from cookie without explicit domain set.

I think I would prefer to have option `cookieDomain` that could be set to an explicit value and that would 1:1 reflect the value passed as cookie option.

Also this solution is not as flexible as explicit domain would be. User might be running sites on `a.foo.bar.com` and `b.foo.bar.com` and want to set cookie domain to `foo.bar.com`.

Reply:
I initially had the same thought, but decided against it, as the cookieDomain can only be the actual host. But I see the argument with domains like `a.foo.bar.com`. I'll adapt it to use the `cookieDomain` property